lib/commands/os/download.ts

```diff
@@ -78,11 +79,13 @@ export default class OsDownloadCmd extends Command {
 		}),
 		version: flags.string({
 			description: stripIndent`
-				exact version number, or a valid semver range,
+				version number (ESR or non-ESR versions),
+				or semver range (non-ESR versions only),
 				or 'latest' (includes pre-releases),
-				or 'default' (excludes pre-releases if at least one stable version is available),
+				or 'default' (excludes pre-releases if at least one released version is available),
 				or 'recommended' (excludes pre-releases, will fail if only pre-release versions are available),
-				or 'menu' (will show the interactive menu)
+				or 'menu' (interactive menu, non-ESR versions),
+				or 'menu-esr' (interactive menu, ESR versions)
```

Comment:
> If users typed any of the combinations below, surely they'd expect the "latest", "recommended" or "default" qualifiers to apply to ESR versions rather than non-ESR versions, even if the help output said that the --esr flag could only be used with --version menu, and even if the CLI printed an error if --esr was used with anything other than --version menu. As it happens, those qualifiers only apply to non-ESR versions as currently implemented in the SDK or API.
>
> - `--esr --version latest`
> - `--esr --version recommended`
> - `--esr --version default`

For all modifiers the version for ESR is the same @pdcastro so there is no conflict really and the `download` command could go with the `--esr` argument as well.